modules/nf-core/strdust/meta.yml

```
@@ -0,0 +1,90 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "STRdust"
+description: Tandem repeat genotyper for long reads
+keywords:
+  - long read
+  - tandem repeats
+  - genomics
+tools:
+  - "STRdust":
+      description: "Tandem repeat genotyper for long reads."
+      homepage: "https://github.com/wdecoster/STRdust/"
+      documentation: "https://github.com/wdecoster/STRdust/blob/main/README.md"
+      tool_dev_url: "https://github.com/wdecoster/STRdust"
+      doi: "10.1101/gr.279265.124 "
+      licence: ["MIT"]
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1', single_end:false ]`
+    - bam:
+        type: file
+        description: Sorted (and preferably phased) BAM/CRAM/SAM file
+        pattern: "*.{bam,cram,sam}"
```

Does it take SAM files?

```suggestion
        description: Sorted (and preferably phased) BAM/CRAM/SAM file
        pattern: "*.{bam,cram,sam}"
```